doc/selectors.rst

```diff
@@ -28,26 +28,28 @@ string patterns in, CadQuery will automatically use the associated selector obje
 Combining Selectors
 ==========================
 
-Selectors can be combined arithmetically and logically, so that it is possible to do intersection, union, and other
-combinations.  For example::
+Selectors can be combined logically, currently defined operators include **and**, **or**, **not** and **exc[ept]** (set difference).  For example:
 
-    box = cadquery.Workplane("XY").box(10,10,10)
+.. cq_plot:: 
```

Comment:
`cadquery.cq_directive` embeds an image of the rendered cq code in the final doc. This is how it renders for me

![cq_directive](https://cloud.githubusercontent.com/assets/13981538/25302673/badd9b50-2743-11e7-8e28-50fb69b5c1df.png)